lib/scripts/repo_export_in_batches.rb

```diff
@@ -1,10 +1,11 @@
-# Before you run this script empty the directory tmp/export/!
-#   rm -rf tmp/export/; mkdir tmp/export/
 # Run this script with the following:
 #   ./bin/rails r lib/scripts/repo_export_in_batches.rb > "$(date +'%Y-%m-%d-%H%M%S')_repo_export_in_batches.log"
-# OR
-#   rm -rf tmp/export/; mkdir tmp/export/; ./bin/rails r lib/scripts/repo_export_in_batches.rb > "$(date +'%Y-%m-%d-%H%M%S')_repo_export_in_batches.log"
-#
+
+puts "---------\nRemoving previous export (if it exists) at tmp/export\n---------"
+export_dir = "tmp/export"
+FileUtils.rm_rf(Dir[export_dir])
+FileUtils.mkdir(export_dir)
```

Comment:
We are probably safer with `mkdir_p` to create `tmp` if it doesn't exist.

Reply:
Good point